Initial commit of labs project moved from Sample Pages repo (#46)
skuid/skuid-labs: experiments/snippets/kanbanBoards/README.md

> # Kanban Boards. 

Reviewer: ```suggestion
# Kanban Boards
```

No periods in titles

Author: Will be fixed in next commit.

> a very popular type of interface for managing lots of objects through a series of phases is the Kanban board.  Perhaps made most popular by Trello. The column arrangement seems very intuitive.  And everyone loves to drag and drop - moving items through the process! 

Reviewer: ```suggestion
Kanban boards are a very popular type of interface for managing lots of objects through a series of phases.  Perhaps made most popular by Trello, the column arrangement is very intuitive for moving items through a process.  And everyone loves to drag and drop!
```

> While Skuid does not offer a Kanban board out of the box - with a little customization and javascript - one can be developed. 

Reviewer: Kanban is not a proper noun, so gonna add suggestions for lower case.
```suggestion
While Skuid does not offer a kanban board out of the box - with a little customization and javascript - one can be developed. 
```

> Here are two Skuid Kanban boards:  

Reviewer: ```suggestion
Here are two Skuid kanban boards:  
```

> 1.  Drag and drop opportunity planning wall. 

Reviewer: Multi-word adjectives need hyphens
```suggestion
1.  Drag-and-drop opportunity planning wall. 
```

Here we see opportunities. They are broken down into columns for each stage.  A snippet enables drag and drop interactions, so an opportunity can be dragged from one column into another - which updates the stage name value.

This example uses the following stage names.  
- Prospecting
- Negotiation/Review
- Proposal/Price Quote
- Closed Won
- Closed Lost

If you have defined different stage names in your org,  you will need to update the conditions on the opportunity models. 

> 2. Task management Kanban.  

Reviewer: Don't need periods in headings.

```suggestion
2. Task management kanban
```
